charon/src/options.rs

```diff
@@ -119,6 +119,10 @@ pub struct CliOpts {
     "))]
     #[serde(default)]
     pub no_code_duplication: bool,
+    /// Monomorphize the code, replacing generics with their concrete types.
+    #[clap(long = "monomorphize")]
+    #[serde(default)]
+    pub monomorphize: bool,
```

Comment:
This changes the serialized AST, therefore please increment the version in `Charon.toml` and run `make test`, this ensures we get nice errors if an ocaml project tries to read an llbc file with the wrong charon version.